connect/runtime/src/main/java/org/apache/kafka/connect/runtime/errors/ProcessingContext.java

```diff
@@ -216,4 +216,10 @@ public void reporters(Collection<ErrorReporter> reporters) {
         this.reporters = reporters;
     }
 
+    @Override
+    public void close() throws Exception {
```

Comment:
Is ```Exception``` in method signature necessary? the method signature of ```ErrorReporter#close``` don't include checked exception.

Reply:
You're right, i've removed the throws annotation here and above. Thanks!